ccdproc/tests/test_combiner.py

```diff
@@ -100,6 +100,16 @@ def test_weights_shape(ccd_data):
     with pytest.raises(ValueError):
         c.weights = ccd_data.data
 
+def test_1Dweights():
+    ccd_list = [CCDData(np.zeros((10, 10)), unit=u.adu),
+                CCDData(np.zeros((10, 10)) - 1000, unit=u.adu),
+                CCDData(np.zeros((10, 10)) + 1000, unit=u.adu)]
+        
+    c = Combiner(ccd_list)
+    c.weights = np.array([1, 5, 10])
+    ccd = c.average_combine()
```

Comment:
So far this ensures that the new weights succeed but we should also check the result. It looks like the result should be 312.5 for the data and weights given, so something like this should do the trick:

```python
assert all(ccd.data == 312.5)
```

If that fails it is probably because of rounding, so you may need to do this instead:

```python
np.testing.assert_almost_equal(ccd.data, 312.5)
```